### ISSUE TITLE
How specify Vocab for language?

### ISSUE THREAD

**Issue by ghost** [NatGr/dc-tts-pytorch#2]
Thanks

**Comment by NatGr** [comment]
Hi, 
the vocabulary is one of the command-line arguments of the train.py and synthsize.py scripts.
The default value is the vocab used by the pre-trained french model. 
If you want to use a pretrained tf model, you should use the same vocabulary as them (see https://github.com/Kyubyong/dc_tts/blob/master/hyperparams.py for english and https://github.com/Kyubyong/css10/blob/master/dc_tts/hyperparams.py for the rest).
If you train from scratch you can fix the vocab to whatever you want.

It is indeed not clear from the README, I'll correct that asap.

**Comment by ghost** [comment]
Thanks you, very help. Do you know how make Arabic vocabulary? Or have AR dataset?

**Comment by NatGr** [comment]
There are tacotron based pre-trained models
https://github.com/youssefsharief/arabic-tacotron-tts

It uses this corpus http://en.arabicspeechcorpus.com/ (I just found both these references now on github, I have no idea what they are worth).

For the vocabulary, you can just take the set of all the chars that occur in the dataset preceded by two special characters (you can use whichever you want but they mustn't be part of the set of chars) to represent "Padding" and "End of Text".

**Comment by ghost** [comment]
Thanks! really helpful. 
When specify vocab, is problem right to left vs left to right?

**Comment by NatGr** [comment]
You're welcome.

Order doesn't matter as long as your vocab starts with "Padding" and "End of Text".
